data/mainnet/tokens.js

```diff
@@ -45,6 +45,25 @@ module.exports = {
     ],
     cmcRewardTokenSymbols: ['FARM', 'WETH'],
   },
+  USDC: {
```

Comment:
Where does this whole entry come from? Why is it here in the file? In my file the USDC entry is near the bottom. Please leave it in the original place.